python_modules/dagster/dagster/core/storage/alembic/versions/015_change_varchar_to_text_sqlite.py

```diff
@@ -0,0 +1,91 @@
+"""change varchar to text
+
+Revision ID: 0da417ae1b81
+Revises: 4ea2b1f6f67b
+Create Date: 2021-01-14 12:29:33.410870
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+from dagster.core.storage.migration.utils import has_table
+
+# pylint: disable=no-member
+
+# revision identifiers, used by Alembic.
+revision = "0da417ae1b81"
+down_revision = "4ea2b1f6f67b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if has_table("event_logs"):
```

Comment:
This script applied the same alembic hash across all 3 sqlite storages (run, event_log, schedule).  This version of the script combines them all but with the `has_table` checks for each respective storage.